devicon.json

```diff
@@ -5628,6 +5628,24 @@
         "color": "#b74af7",
         "aliases": []
     },
+    {
+        "name": "playwright",
```

Comment:
```suggestion
        "name": "playwright",
        "altnames": [],
```

Reply:
alnames is missing

Reply:
Hi, I have added the altnames as you mentioned above. Thank you for your feedback.